templates/_site/_snippet/_content/_blocks/_image.twig

```diff
@@ -64,7 +64,7 @@
                                     {{ image.imageCaption }}
                                 {% endif %}
                                 {% if image.imageCopyright|length and image.imageCaption|length %}
```

Comment:
Lijn 66 t.e.m. 68 zouden weg mogen, en de © mag verschijnen in de conditional van lijn 69.